osm.py

```diff
@@ -536,7 +536,7 @@ def diff_settings(dest, src):
 
 
 def backup_vault(vault_path):
-    print('#', vault_path)
+    print(f'# Backing up settings for: {vault_path}')
```

Comment:
I wasn't sure about keeping this as `print` vs `verbose` but when I was testing it I liked seeing that it was doing something and where, but I'm open to making this `verbose` instead.